index.html

```diff
@@ -16,17 +18,24 @@
 <body>
   <header>
   <nav>
-    <ul class="menu">
-        <div>
-          <li class="logo"><a href="">My Logo</a></li>
-        </div>
-        <div class="menu2">
-          <li class="item"><a href="#">Portfoilo</a></li>
-          <li class="item"><a href="#">about</a></li>
-          <li class="item"><a href="#">contact</a></li>
-          <li class="toggle"><a href=""><img src="./images/Icon - Menu.png" alt="humberger menu image"></a></li>
-        </div>
-    </ul>
+    <div class="menu">
+          <div class="logo"><a href="#">My Logo</a></div>
+        <ul class="desktop-nav st1">
+          <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-link"><a href="#about">about</a></li>
+          <li class="nav-link"><a href="#contact">contact</a></li>
```

Comment:
As requested by the previous reviewer, kindly adjust the font face of your text. Looking at the template on Figma, the text should be capitalized. 👀👍i.e 
- Portfolio
- About
- Contact

Your design
![image](https://user-images.githubusercontent.com/78251927/233395787-a090e351-88a7-409c-aaa5-0d4c277dc2bc.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/233396032-b144de87-4572-412d-bdd7-aa62a4985e39.png)